model/src/main/java/org/cbioportal/model/AlterationCountByGene.java

```diff
@@ -8,9 +8,9 @@
 
     private Integer entrezGeneId;
     private String hugoGeneSymbol;
-    private Integer numberOfAlteredCases;
+    private Integer numberOfAlteredCases = 0;
```

Comment:
I don't think this is the ideal way of assigning a default value. It might be better/safer to do this in the function (or sql query) where we generate/map the actual data. Our models should be just a simple representation of the data without any initialization logic.